static/scripts/main.js

```diff
@@ -6,23 +6,39 @@ window.onload = function () {
     if(hash) {
       hash.scrollIntoView()
     }
+    $('[data-md-sticky-header]').headroom({
+      offset: UnpinOffset,
+      tolerance: 0
+    });
 }
 
-var CurrentScroll = 0;
+var SearchbarTop = 61;
```

Comment:
It is the topmost vertical position of the searchbar. I agree we shouldn't be using a magic value for that, we can compute it dynamically. I'll look into that, I think $(".search-wrapper").offset().top will do.